packages/replayio/src/utils/prompt/prompt.ts

```diff
@@ -39,9 +40,9 @@ export async function prompt({
     }
 
     stdin.on("data", onData);
-
-    if (abortSignal) {
-      abortSignal.addEventListener("abort", destroy);
-    }
+    signal?.addEventListener("abort", () => {
+      destroy();
+      resolve(false);
+    });
```

Comment:
You're now adding something different than what you're removing above.

Reply:
🤦 good catch!